vimeo-networking/src/main/java/com/vimeo/networking/Configuration.java

```diff
@@ -101,27 +101,27 @@ public Cache getCache() {
      * Persist the {@link VimeoAccount} in the {@link AccountStore}
      * <p>
      * Account name is <b>required</b> for authenticated user accounts; for
-     * client credential account saving use {@link #saveAccount(VimeoAccount)}
+     * client credential account saving use {@link #saveNonUserAccount(VimeoAccount)}
      *
      * @param account     the account to persist
      * @param accountName the name of the account.
      */
-    public void saveAccount(@NotNull VimeoAccount account, @NotNull String accountName) {
+    public void saveAuthenticatedUserAccount(@NotNull VimeoAccount account, @NotNull String accountName) {
         if (accountStore != null) {
```

Comment:
Hey, I'm sorry to bring this one up, Kevin... I feel like we've opened a whole can of worms with this PR as we touch different files. This one also still has the old member variable naming (i.e. accountStore vs mAccountStore). If it is too much of a pain in the ass, we can just ticket the rest of the renaming of the members of networking.